packages/docs-reanimated/docs/guides/contributing.mdx

```diff
@@ -0,0 +1,318 @@
+
+#### Example with a video
+
+```javascript
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import AnimatedKeyboardSrc from '!!raw-loader!@site/src/examples/AnimatedKeyboard';
+
+<InteractiveExample
+  src={AnimatedKeyboardSrc}
+  component={
+    <ThemedVideo
+      center
+      width={300}
+      sources={{
+        light: '/recordings/useAnimatedKeyboard_light.mov',
+        dark: '/recordings/useAnimatedKeyboard_dark.mov',
+      }}
+    />
+  }
+/>;
+```
+
+#### Example with an interactive component
+
+```javascript
+import DecayTrain from '@site/src/examples/DecayTrain';
+import DecayTrainSrc from '!!raw-loader!@site/src/examples/DecayTrain';
+
+<InteractiveExample
+  src={DecayTrainSrc}
+  component={<DecayTrain />}
+  label="Grab and drag the train"
+/>;
+```
+
+## Contributing to Code
+
+Posting Pull Requests to the issues is great way to contribute to Reanimated. If you eager to start contributing right away, we have list of [good first issue](https://github.com/software-mansion/react-native-reanimated/pulls?q=is%3Apr+is%3Aopen+label%3A%22good+first+issue%22) that contain bugs which have limited scope. In this section we'll describe in more details how to play around with Reanimated setup.
+
+:::caution
+
+Make sure that you have [React Native setup](https://reactnative.dev/docs/environment-setup) properly and you got familiar with [How to Open a Pull Request](https://reactnative.dev/contributing/how-to-open-a-pull-request) before reading this section further.
+
+:::
+
+We have shared code of example app, that is run on different platforms and architectures. When working on an issue the most important destinations will be:
+
+- `apps/common-app` &ndash; shared source code with example app
+- `packages/react-native-reanimated/src/reanimated2` &ndash; containing source code related to JS part of Reanimated features
+- `packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated` &ndash; containing source code related to Android native part of Reanimated features
+- `packages/react-native-reanimated/apple` &ndash; containing source code related to iOS native part of Reanimated features
+
+Our workflow usually starts from editing `app/src/examples/EmptyExample.tsx` with testing example for our new feature or a bug. This is the part that we often attach to PR so other developers can test the same use-case. Next, we run the `Example` (or `FabricExample` depending on the architecture) app and start development.
+
+:::info
+
+If you want implement new feature or fix bug, but still feel uninformed, after reading through this section, or could use a guidance you can reach Reanimated Team on [discord](https://discord.com/channels/464786597288738816/1216675825584181330).
+
+:::
+
+### Working on Android
+
+To start with, let's install all dependencies:
+
+1. `yarn && yarn build`
+2. `cd apps/common-app && yarn && cd ..`
+3. `cd paper-example && yarn`
```

Comment:
As far as I know we don't need to use `yarn` in neither `common-app` nor `paper-example` anymore because we have hoisted deps from monorepo structure (but please confirm it @tjzel )

Reply:
fixed in https://github.com/software-mansion/react-native-reanimated/pull/6061/commits/7a16a7185d4c979ad1aeacdb8f8398652f07d82d